package.json

```diff
@@ -755,5 +760,6 @@
       "@metamask/solana-wallet-snap>@solana/web3.js>bigint-buffer": false
     }
   },
-  "packageManager": "yarn@4.5.1"
+  "packageManager": "yarn@4.5.1",
+  "foundry": "nightly-31dd1f77fd9156d09836486d97963cec7f555343"
```

Comment:
So groossssss

Reply:
:cry: there's not much we can do about it... it doesn't seem they have semantic versioning? :thinking: